firmware/vcv_plugin/internal/make_element.cc

```diff
@@ -486,6 +486,14 @@ Element make_element(rack::app::SvgButton *widget) {
 	return make_element(widget->sw);
 }
 
+Element make_element(rack::ui::TextField *widget) {
+	DynamicTextDisplay element{};
+	element.text = widget->text;
+	element.font = "Segment32";
+	element.color = Colors565::Red;
```

Comment:
looks like this: 

https://github.com/user-attachments/assets/98dc38a6-6e06-44b7-a555-f769384e75f2